src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -75,8 +75,8 @@ const Sidebar: React.FC<SideBarProps> = ({ open, handleDrawerClose }) => {
         {linkItems.map((linkItem) => (
```

Comment:
also make sure to wrap these linkItems in a <Box> </Box>

Reply:
Sorry, wrap the linkitems in a what?

Reply:
`<Box>` sorry github it weird ab putting code in comments